handbook/ce/team-culture.md

```diff
@@ -0,0 +1,180 @@
+    - Customer Name:
+    - Link to issue (slack / GH / ZD ticket #):
+    - Background information or anything we should know that would be helpful?:
+  - CE may also need to share additional context around an issue with the head of support. For example, a CE may need to share special circumstances around a customer issue. In scenarios such as this, the CE should raise a request in the support channel (via the Support Ticket Request shortcut).  For each request the CE should provide the following information:
+    - Customer Name:
+    - Link to issue (Slack / Github issue / ZD ticket #):
+    - What’s the ask:
+    - Why are we making this ask:
+    - What happens if we don’t take this desired action:
+    - Is there a time consideration we need to be aware of, if so what is it and why?:
+    - Anything else we should know that would be helpful?:
+
+### Software Engineers (SWEs)
+- **How CE adds value to SWEs:** CE provides important insights from prospective and current customers which inform and serve as important inputs to SWE roadmap.
+- **How SWEs adds value to CE:** SWEs create a high quality product and when needed, helps educate CEs on how certain features work so that they can educate customers. SWEs also conduct via planned training sessions, periodic pairing, deep-dives on new features/products, etc.
+- **Collaboration overview:** CEs can pose how-to questions and provide feedback via Slack
+
+### Product
+- **How CE adds value to product:** CEs provide a data-driven view into customer concerns/questions, ad-hoc feedback shared whilst helping customers, and helps update documentation so customers (and we) have a single source of truth.
+- **How product adds value to CE:** Product educates CE on new features and helps clarify expected behavior questions.
+- **Collaboration overview:** CEs provide Product with feedback and feature requests; Product seeks inputs from customer through CEs.
+
+### Marketing
+- **How CE adds value to marketing:** CEs help ensure that when developers think of Sourcegraph, they associate it with quality and responsiveness.
```

Comment:
CEs help ensure marketing material is relevant and valuable to engineers and engineering management